src/components/reportingCards/ReportingCard1.js

```diff
@@ -15,6 +15,25 @@ export default function ReportingCard1(props) {
     rows
   );
 
+  const formatData = (data) => {
```

Comment:
Rather than duplicating this function across reporting cards, I'd suggest creating a shared utility function that replaces blank entries with "No organization" and giving it a name that indicates that modification.

To be more general, it could take the string you want to use in place of blank entries as an input.

Reply:
That was what I originally went with, but there could be other reporting cards later as well. There is a card with gender, and species. But I see you point. We can change the reporting card itself to accept a default value that is displayed if the entry is empty.

Reply:
@nmcharlton For now I've just shifted the formatData function to utilities. It would be better to refactor the reporting cards in a separate issue.